### ISSUE TITLE
NGAlert/LegacyAlert: State Annotations are not colored correctly

### ISSUE THREAD

**Issue by kylebrandt** [grafana/grafana#37304]
**What happened**:
When creating an alert in the new system that is linked to panel, the annotations for state changes do not have corresponding colors like they did in dashboard alerting, so it is hard to follow:

![image](https://user-images.githubusercontent.com/1692624/127326067-a3b25a5d-1644-45bd-8918-c96a9858befd.png)

That image has "Pending", "Normal" and "Alerting" annotations. (Note: Technically they are ` Normal` with a prefixed space, but when I edit them the color stays the same, can look into fixing that when we know what the frontend is expecting.

**What you expected to happen**:

Them to be colored as before (need to check colors, I think red = alerting, yellow = pending, green = normal). 

**How to reproduce it (as minimally and precisely as possible)**:
Create a Grafana managed alert rule in the new NG system that is linked to dashboard via the Dashboard UID and Panel ID annotations on the rule (which are populated automatically if you create the rule via the dashboard UI).

**Anything else we need to know?**:
Not sure if this is frontend and/or backend yet. The are some field in the db like (prev_state, new_state, alert_id), but I have checked what the old system makes yet to see if shows up correctly when I edit the db. 

**Environment**:
- Grafana version: ddf5b65c51
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser:
- Grafana plugins:
- Others:

**Comment by kylebrandt** [comment]
Note: Looking into this more, seems to be true of annotations when `ngalert` is _not_ enabled as well.

**Comment by cmckeen** [comment]
Hey @kylebrandt, we are seeing this behavior as well.  Furthermore, on a dashboard with multiple panels, the annotations are replicated across all panels.  We are running [v8.1.0-beta2](https://github.com/grafana/grafana/releases/tag/v8.1.0-beta2).  Please let me know if I should open a separate issue for that.

![Screen Shot 2021-08-05 at 9 23 31 AM](https://user-images.githubusercontent.com/866968/128357501-6a5029ed-8c07-4eb1-8c83-548eca4ade07.png)

**Comment by underdog** [comment]
I'm running Grafana v8.1.0-beta3 (f778fdbbe8) and still see this occurring.

**Comment by underdog** [comment]
I converted over to ngalert.
appears that the annotations are working properly, but the coloring is still an issue, which I think is still under #37412
![image](https://user-images.githubusercontent.com/510960/128397353-4128cd18-2fb4-469a-87e1-b1dc39ce1aba.png)
